modules/apps/transfer/keeper/relay_test.go

```diff
@@ -149,16 +157,95 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestSendTransferSetsTotalEscrowAmountForSourceIBCToken() {
```

Comment:
Yes, I could look into that, but to be honest in my opinion the `TestSendTransfer` is a bit overloaded already (I was actually thinking of opening a PR with a suggestion to split it, since every time I have to come back to it I have a hard time understanding the tests cases)...